TTS/utils/manage.py

```diff
@@ -0,0 +1,103 @@
+import json
+import gdown
+from pathlib import Path
+import os
+
+from TTS.utils.io import load_config
+
+
+class ModelManager(object):
+    """Manage TTS models defined in .models.json.
+    It provides an interface to list and download
+    models defines in '.model.json'
+
+    Models are downloaded under '.TTS' folder in the user's
+    home path.
+
+    Args:
+        models_file (str): path to .model.json
+    """
+    def __init__(self, models_file):
+        super().__init__()
+        self.output_prefix = os.path.join(str(Path.home()), '.tts')
```

Comment:
I would recommend to store data in the appropriate application directories instead of `.tts`:

```python
import sys
from pathlib import Path
from os import getenv

def get_user_data_dir(appname):
    if sys.platform == "win32":
        import winreg
        key = winreg.OpenKey(
            winreg.HKEY_CURRENT_USER,
            r"Software\Microsoft\Windows\CurrentVersion\Explorer\Shell Folders"
        )
        dir_,_ = winreg.QueryValueEx(key, "Local AppData")
        ans = Path(dir_).resolve(strict=False)
    elif sys.platform == 'darwin':
        ans = Path('~/Library/Application Support/').expanduser()
    else:
        ans=Path(getenv('XDG_DATA_HOME', "~/.local/share")).expanduser()
    return ans.joinpath(appname)
```

Reply:
I don't run things on windows so I don't know how it works on it but Path.home() should set somewhere reasonable on any platform.  Are you on Win?

Reply:
`home()` works on windows but there is not concept of prefixing directories with a `.` in order to hide them. This means it would be just shown as a normal directory and litter the user home.

Reply:
Also for the other operating systems it is nicer to stick to the standards. On Linux and friends this is XDG: https://www.freedesktop.org/wiki/Specifications/

Reply:
Here you can see the different locations chosen for each OS: https://github.com/Mic92/test-actions/actions/runs/512669735

Reply:
now I adapted your code